js/src/setup-mc/setup-stepper/setup-free-listings/shipping-rate/shipping-rate-setup/index.js

```diff
@@ -18,10 +11,7 @@ import AppSpinner from '.~/components/app-spinner';
 import useTargetAudienceFinalCountryCodes from '.~/hooks/useTargetAudienceFinalCountryCodes';
 import './index.scss';
 
-const ShippingRateSetup = ( props ) => {
-	const {
-		formProps: { getInputProps, values },
-	} = props;
```

Comment:
💅 

If the `formProps` prop would not be used in the subsequent PRs, we could consider removing it from the upper layer.

https://github.com/woocommerce/google-listings-and-ads/blob/3c9e134675d914750498c1def7fb2259328c61d5/js/src/setup-mc/setup-stepper/setup-free-listings/shipping-rate-section.js#L118

Reply:
Good catch! ShippingRateSetup will be removed in https://github.com/woocommerce/google-listings-and-ads/pull/1267, specifically in commit [`6135d95` (#1267)](https://github.com/woocommerce/google-listings-and-ads/pull/1267/commits/6135d95cb9f86553d2ecf2c252cde11c471d9dd1), but to make things cleaner in this PR, I have fixed it in [`77c877d` (#1263)](https://github.com/woocommerce/google-listings-and-ads/pull/1263/commits/77c877d0e5740f411fae082098edf28cdee2bb4c).

Reply:
Got it, thanks!